sites/docs/src/pages/_ViewConfigEditor.js

```diff
@@ -114,11 +120,27 @@ export default function ViewConfigEditor(props) {
     setUrl(nextUrl);
   }
 
+  async function handleConfigGeneration() {
+    const urls = datasetUrl.split(/, | |,/);
```

Comment:
Can we use semi-colon (trimming any whitespace at URL start/end) instead of space/comma?

Reply:
yes, of course. Done